main.py

```diff
@@ -157,42 +142,112 @@ def add_menu(self):
+                                      ["2. Додати нотатку"],
+                                      ["3. Знайти нотатку"],
+                                      ["4. Змінити нотатку"],
+                                      ["5. Видалити нотатку"],
+                                      ["6. Повернутись в попереднє меню"]])
+        return show_notes_menu
+
+
+class Handler:
     def __init__(self):
+        self.menu = Menu()
+        self.notes_book = NotesBook()
+        self.address_book = AddressBook()
 
-        self.Adderessbook = AddressBook()
-        self.notes = []  # a func add_note should create an instance of Note
-        # and add it to current list
-        self.menu = Menu()  # should be changed
+    def action_note(self, notes_book: NotesBook):
+        self.notes_book = notes_book
+        while True:
+            print(self.menu.notes_menu)
+            action = input("\033[34m" + "Обери потрібну команду(1-5), "
+                                         "або я спробую вгадати: ")
+            if action.lower() in ["1", "check", "подивитись", "нотатки",
+                                     "замітки", "заметки"]:
+                print(self.notes_book.data)
+            elif action.lower() in ["2", "create", "створити", "создать",
+                                     "замітки", "заметки"]:
+                note = Note(input('Введіть нонатку: '))
+                self.notes_book.add_note(note)
+            elif action.lower() in ["3", "знайти", "search", "нотатки",
+                                     "замітки", "заметки"]:
+                pass
+                #self.notes_book.add_note(note)
+            elif action.lower() in ["exit", "close", "good bye", "5", "вихід",
+                                    "выход"]:
+                break
+            else:
+                print('You was wrong or notes didn\'t create')
+
+    def action_phone(self, address_book: AddressBook):
+        pass
 
+    def main_action(self):
         while True:
             print(self.menu.main_menu)
-            command = input("I am waiting for your command: ")
 
-            if command.lower() in ["exit", "close", "good bye", "11", "вихід", "выход"]:
+            command = input("\033[34m" + "Обери потрібну команду(1-5), "
+                                         "або я спробую вгадати: ")
+            if command.lower() in ["exit", "close", "good bye", "5", "вихід", "выход"]:
                 print("Good bye!")
+                return 'break'
+            elif command.lower() in ["нотатки", "note", "notes", "2",
+                                     "замітки", "заметки"]:
+                self.action_note(self.notes_book)
+            elif command.lower() in ["phone", "телефон", "номер", "1","number"]:
+                self.action_phone(self.address_book)
+
+
+class Bot:
+
+    def __init__(self):
+        self.menu = Menu()  # should be changed
+        self.handler = Handler()
+        while True:
```

Comment:
No reason to use cycle here. You will already start endles cycle by calling   self.handler.main_action()(and it will stop with exit return)